src/scenarios/billing/UnsubscribeFromSubscriptionPlan/graphql-documents.ts

```diff
@@ -0,0 +1,28 @@
+import gql from 'graphql-tag';
+
+export const getAllSubscriptionsQuery = gql`
+  query GetSubscriptions {
+    subscriptions(orderBy: LIFECYCLE_STATUS_ASC) {
+      nodes {
+        id
+        lifecycleStatus
+        subscriptionPlan {
+          title
+        }
+        paymentPlan {
+          title
+        }
+      }
+    }
+  }
+`;
+
+export const CancelSubscriptionMutation = gql`
+  mutation CancelSubscription($input: PaypalCancelSubscriptionInput!) {
+    paypalCancelSubscription(input: $input) {
+      subscription {
+        lifecycleStatus
```

Comment:
let's also add the the subscription id to the query

Reply:
Done